Documentation/Input/Overview.md

```diff
@@ -5,14 +5,18 @@ The Input System in MRTK allows you to:
 - Define abstract actions, like *Select* or *Menu*, and associate them to different inputs.
 - Setup pointers attached to controllers to drive UI components via focus and pointer events.
 
-Inputs are produced by [**Input Providers**](InputProviders.md). Each provider corresponds to a particular source of input: Open VR, Windows Mixed Reality (WMR), Unity Joystick, Windows Speech, etc. Providers are added to your project via the **Registered Service Providers Profile** in the *Mixed Reality Toolkit* component and will produce [**Input Events**](InputEvents.md) automatically when the corresponding input sources are available, e.g. when a WMR controller is detected or a gamepad connected.
+<img src="../../Documentation/Images/Input/MRTK_InputSystem.png" style="display:block;margin-left:auto;margin-right:auto;">
+<sup>Overview of MRTK Input System</sup>
```

Comment:
Nit: It's a bit weird seeing intermixtures of markdown and HTML (i.e. sup vs ##) - might be worth sticking to one particular one (also, using the markdown thing creates a link to the specific header section which is nice).

Maybe sup does the same thing though.

Reply:
Huh, I guess it does. It might be worth still being consistent with markdown (i.e. use ## here if only because the top level is #)

---
In reply to: [296307767](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/5000#discussion_r296307767) [](ancestors = 296307767)

Reply:
For the images, I had to use <img> tags instead of markdown to support specific width. (somehow, image width is not supported by pure markdown) For the <sup> I found it from other pages used for the image caption purpose. It seems like <figure> and <figcaption> tags are correct ones. https://www.w3schools.com/TAGS/tag_figcaption.asp 
I'll test it out.

Reply:
<figure and <figcaption tags are not working in github. I'll keep using <img with <sup for the image captions.